src/ansys/fluent/core/utils/dictionary_operations.py

```diff
@@ -1,6 +1,30 @@
 """Performs some operations on Python dictionaries."""
 
+from typing import Any
 
-def get_first_dict_key_for_value(input_dict, value):
-    """Get the first dictionary key that matches a value."""
-    return next((key for key, val in input_dict.items() if val == value), None)
+
+def get_first_dict_key_for_value(input_dict: dict, value: Any):
+    """Get the first dictionary key that matches a value. Typical
+    usage is where the value is known to be unique in the input
+    dictionary.
+
+    Parameters
+    ----------
+    input_dict : dict
+    value : Any
+
+    Returns
+    -------
+    Any
+        The key associated with the first match.
+
+    Raises
+    ------
+    ValueError
+        If the value is absent from the dictionary.
+    """
+
+    try:
+        return next((key for key, val in input_dict.items() if val == value))
```

Comment:
I removed the default-to-`None` strategy, which doesn't work because `None` is a valid key.